api/openapi-spec/swagger.json

```diff
@@ -4236,7 +4236,7 @@
      {
       "uniqueItems": true,
       "type": "string",
-      "description": "name of the Pod",
+      "description": "name of the PodAttachOptions",
```

Comment:
This really should be "name of the Pod". This is an existing problem, for example, the doc for replicaset/scale also shows "name of the Scale" while it should be "name of the replicaSet".